src/frontend/components/DashboardThumbnail/index.spec.tsx

```diff
@@ -389,9 +376,16 @@ describe('<DashboardThumbnail />', () => {
     expect(
       useThumbnail.getState().thumbnails[thumbnailCreated.id],
     ).not.toBeDefined();
-    render(
-      wrapInIntlProvider(<DashboardThumbnail video={videoWithoutThumbnail} />),
-    );
+    render(<DashboardThumbnail video={videoWithoutThumbnail} />, {
+      routerOptions: {
+        routes: [
+          {
+            path: '/form/thumbnails/42',
+            render: () => <span>form thumbnails</span>,
+          },
+        ],
+      },
```

Comment:
I'm not sure to like it neither… :/
Maybe we could call it `fakeRouterOptions`, or `mockedRouterOptions` ? This way, it's clear that it is related to testing only ?